extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -25,69 +30,113 @@ const TrackError = memo( ( { link } ) => (
+			>
+				<a
+					className="jetpack-podcast-player__episode-link"
+					href={ track.link }
+					role="button"
+					aria-pressed="false"
+					onClick={ e => {
+						// Prevent handling clicks if a modifier is in use.
+						if ( e.shiftKey || e.metaKey || e.altKey ) {
+							return;
+						}
 
-					// Select track.
-					selectTrack( index );
-				} }
-				onKeyDown={ e => {
-					// Only handle the Space key.
-					if ( event.key !== ' ' ) {
-						return;
-					}
+						// Prevent default behavior (opening a link).
+						e.preventDefault();
 
-					// Prevent default behavior (scrolling one page down).
-					e.preventDefault();
+						// Select track.
+						selectTrack( index );
+					} }
+					onKeyDown={ e => {
+						// Only handle the Space key.
+						if ( event.key !== ' ' ) {
+							return;
+						}
 
-					// Select track.
-					selectTrack( index );
-				} }
-			>
-				<span className="jetpack-podcast-player__episode-status-icon"></span>
-				<span className="jetpack-podcast-player__episode-title">{ track.title }</span>
-				{ track.duration && (
-					<time className="jetpack-podcast-player__episode-duration">{ track.duration }</time>
-				) }
-			</a>
-			{ isActive && isError && <TrackError link={ track.link } /> }
-		</li>
-	);
-} );
+						// Prevent default behavior (scrolling one page down).
+						e.preventDefault();
+
+						// Select track.
+						selectTrack( index );
+					} }
+				>
+					<span className="jetpack-podcast-player__episode-status-icon"></span>
+					<span className="jetpack-podcast-player__episode-title">{ track.title }</span>
+					{ track.duration && (
+						<time className="jetpack-podcast-player__episode-duration">{ track.duration }</time>
+					) }
+				</a>
+				{ isActive && isError && <TrackError link={ track.link } /> }
+			</li>
+		);
+	}
+);
 
-const Playlist = memo( ( { tracks, selectTrack, currentTrack, playerState } ) => {
-	return (
-		<ol className="jetpack-podcast-player__episodes">
-			{ tracks.map( ( track, index ) => (
-				<Track
-					key={ track.id }
-					index={ index }
-					track={ track }
-					selectTrack={ selectTrack }
-					isActive={ currentTrack === index }
-					isError={ currentTrack === index && playerState === STATE_ERROR }
-				/>
-			) ) }
-		</ol>
-	);
-} );
+const Playlist = memo(
+	( {
+		tracks,
+		selectTrack,
+		currentTrack,
+		playerState,
+		primaryColor,
+		customPrimaryColor,
+		secondaryColor,
+		customSecondaryColor,
+	} ) => {
+		return (
+			<ol className="jetpack-podcast-player__episodes">
+				{ tracks.map( ( track, index ) => (
+					<Track
+						key={ track.id }
+						index={ index }
+						track={ track }
+						selectTrack={ selectTrack }
+						isActive={ currentTrack === index }
+						isError={ currentTrack === index && playerState === STATE_ERROR }
+						primaryColor={ primaryColor }
+						customPrimaryColor={ customPrimaryColor }
+						secondaryColor={ secondaryColor }
+						customSecondaryColor={ customSecondaryColor }
```

Comment:
Just a thought. This number of props on this component is getting quite large.

I wonder whether a single `colors` prop would be better.

Something like

```js
{
    'primary': {
        'default': '',
        'custom': '',
    }
    'secondary': {
        'default': '',
        'custom': '',
    }
}
```

Reply:
sounds good to me. Let me update the code.